rl4co/models/zoo/matnet/model.py

```diff
@@ -26,9 +26,9 @@ def __init__(
         if kwargs.get("num_augment", 0) != 0:
             log.error("MatNet does not use symmetric augmentation. Setting num_augment to 0.")
         kwargs["num_augment"] = 0
-        if kwargs.get("use_dihedral_8", True):
+        if kwargs.get("augment_fn") != 'symmetric':
             log.error("MatNet does not use symmetric Dihedral Augmentation. Setting use_dihedral_8 to False.")
-        kwargs["use_dihedral_8"] = False
+        kwargs["augment_fn"] = 'symmetric'
```

Comment:
MatNet does not support `symmetric` (because space is non-euclidean) so this one should not be set